### PR TITLE
Update ng to v8

### PR DESCRIPTION
**Pull request by MichalMartinek**
Fixed also other issues - test and lint errors

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -11,32 +11,32 @@
...
-    "@angular/common": "^7.0.4",
-    "@angular/compiler": "^7.0.4",
-    "@angular/core": "^7.0.4",
-    "@angular/forms": "^7.0.4",
+    "@angular/animations": "^8.1.2",
+    "@angular/common": "^8.1.2",
+    "@angular/compiler": "^8.1.2",
+    "@angular/core": "^8.1.2",
+    "@angular/forms": "^8.1.2",
     "@angular/http": "^7.0.4",
```
Comment (dborosova) [inline package.json]: Balicek `@angular/http` je deprecated. Pokud se objevuje v kodu, mel by byt nahrazen za `@angular/common/http`

Comment (MichalMartinek) [reply package.json]: Balíček jsem odstranil, ani se nepoužíval.

**Comment by MichalMartinek** [comment]
@MichalMaM I used the command `ng update @angular/cli @angular/core` but it failed in the procces, so i did the rest of the update manually. After rerun it looks ok.

**Comment by MichalMartinek** [comment]
JQuery is required for the bootstrap, so I replaced it with `bootstrap-css-only`.

**Pull request merged**